backend/blog/admin.py

```diff
@@ -31,6 +31,6 @@ class PostAdmin(AdminUsersMixin):
     user_fk = "author_id"
 
     def author_display_name(self, obj):
-        return self.get_user_display_name(obj.author_id)
+        return self.get_user_data(obj.author_id)["displayName"]
```

Comment:
why are you changing this? 🤔

Reply:
thought was better having one method to get the data... but maybe not?

Reply:
get_user_display_name is a utility method that calls get_user_data so we don't have to use ["displayName"] everywhere since that will be the most common use.

get_user_data is what you have to use since you have to access the gender field.